src/content/docs/how-to-setup-freecodecamp-locally.mdx

```diff
@@ -52,7 +52,57 @@ There are various ways to launch a Gitpod workspace:
 
 </Steps>
 
-That's it, you can now skip to the 'syncing up from parent' section after you have launched a Gitpod workspace. Most parts of this guide applies to Gitpod workspaces, but be mindful of [how the URLs & Ports work within a Gitpod](https://www.gitpod.io/docs/configure/workspaces/ports) workspace.
+#### Running node tests in Gitpod
+
+Once you have a Gitpod instance running, open a terminal and then install the following
+packages:
+
+```bash
+sudo apt update
+sudo apt-get install -y \
+        ca-certificates \
+        fonts-liberation \
+        libappindicator3-1 \
+        libatk-bridge2.0-0 \
+        libatk1.0-0 \
+        libc6 \
+        libcairo2 \
+        libcups2 \
+        libdbus-1-3 \
+        libexpat1 \
+        libfontconfig1 \
+        libgbm1 \
+        libgcc1 \
+        libglib2.0-0 \
+        libnspr4 \
+        libpango-1.0-0 \
+        libpangocairo-1.0-0 \
+        libstdc++6 \
+        libx11-6 \
+        libx11-xcb1 \
+        libxcb1 \
+        libxcomposite1 \
+        libxcursor1 \
+        libxdamage1 \
+        libxext6 \
+        libxfixes3 \
+        libxi6 \
+        libxrandr2 \
+        libxrender1 \
+        lsb-release \
+        wget \
+        xdg-utils
+```
```

Comment:
Why does the use have to do all this? Can this not be baked into the the Dockerfile?

Reply:
I assume it can, but I think it adds to the startup time. I vaguely recall that it's quite slow.

@ShaunSHamilton do you remember if that's correct?